From e90a9bb36dc07246d49d5fe9ee693c7e22289847 Mon Sep 17 00:00:00 2001
From: Peter Volkov <pva@gentoo.org>
Date: Fri, 10 Sep 2010 11:19:32 +0400
Subject: [PATCH 2/2] Update udev rules to match current specification

Udev-151 issues following warning:

udevd[569]: NAME="%k" is superfluous and breaks kernel supplied names,
please remove it from /etc/udev/rules.d/60-vzctl.rules:1

This patch fixes it.

Original report: http://bugs.gentoo.org/show_bug.cgi?id=335762
---
 etc/udev/60-vzctl.rules |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/etc/udev/60-vzctl.rules b/etc/udev/60-vzctl.rules
index 95cb609..e44fec5 100644
--- a/etc/udev/60-vzctl.rules
+++ b/etc/udev/60-vzctl.rules
@@ -1 +1 @@
-KERNEL=="vzctl", NAME="%k", MODE="0600"
+KERNEL=="vzctl", MODE="0600"
-- 
1.7.2.2

